platform/windows/display_server_windows.cpp

```diff
@@ -525,6 +525,24 @@ void DisplayServerWindows::delete_sub_window(WindowID p_window) {
 		wintab_WTClose(windows[p_window].wtctx);
 		windows[p_window].wtctx = 0;
 	}
+
+	// we only want this to happen if we close an active/focused window
+	if (GetActiveWindow() == windows[p_window].hWnd) {
+		Map<WindowID, WindowData>::Element *E = windows.back(); // get highest WindowID
+		if (E->key() == p_window) { // we want the highes WindowID that is not p_window
```

Comment:
the code tells us what is in the the comments and therefor you can omit them

Reply:
Everything squashed and the two comments removed.

I just noticed clang-format might have resorted the indludes... does it matter?

Reply:
yes it does, thats why the travis check is failing

Reply:
please don't remove comments, to me that piece of source code looks like complete gibberish
even then, "get the highest window id" doesn't mean anything to me, but at least it's an explanation

you guys are assuming that everybody who reads the engine source code understands everything 100% which is plain wrong, and it makes working with the engine as a first timer a pain, my only option is to literally break the engine and to see what happens

Reply:
Well, map.back() is implemented a certain way and returns the highest window, because it is sorted by the WindowID. The problem with comments is, that they are not compile. I understand the problem you are facing, but the DisplayServers tend to be less readible, because of the stuff that has to be done in order to communicate with the operating system.